Source/WebCore/accessibility/AccessibilityTree.cpp

```diff
@@ -82,23 +82,26 @@ bool AccessibilityTree::isTreeValid() const
     Node* node = this->node();
     if (!node)
         return false;
+
+    // This function does not require smart pointers as long as we ensure it does not modify the DOM
+    ASSERT(ScriptDisallowedScope::InMainThread::isEventDispatchAllowedInSubtree(node));
```

Comment:
No need for this comment, since you've added the RefPtrs.

Reply:
Oops, good catch!